### DISCUSSION TITLE
Pairing issues after upgrading from 2531 to Sonoff Zigbee 3.0 USB plus usb

### DISCUSSION THREAD

**Discussion by joshdinsdale** [Koenkk/zigbee2mqtt#11297] [Devices]
Hi All

I've recently upgraded my stick to the Sonoff USB plus. Im having a couple of odd issues:

I have 2 tradfri E1603/E1702/E1708 sockets, neither reconnected after the change of stick. With the first one, i made the mistake of trying to repair it using its standard pairing mechanism, whatever i try now I cannot get it to repair (with the second socket i just powered it off and back on and it reconnected fine..)
I also have a Sonoff SNZB-04 contact sensor that will not reconnect either.

Now, when the stick arrived i upgraded it to the the latest co-ordinator firmware (Z-Stack_Home_3.x.0_20211217), however i forgot to change the ieee address. I thought this might have caused the issues, so i reflashed and added the option to change the address (i had already made a note of this previously).

Anyway, i still cant get the Tradfri socket or Sonoff contact sensor to repair, i get nothing in the logs, i've tried them both right next to the stick to try to elimate any routers but doesnt seem to help.

Does anyone have any other ideas I could try?

Thanks
Josh

**Comment by Koenkk** [comment]
see https://github.com/Koenkk/zigbee2mqtt/issues/10339
